mininterface/textual_interface/widgets.py

```diff
@@ -111,6 +116,7 @@ class MySecretInput(Horizontal, Changeable):
     def __init__(self, tag: SecretTag, **kwargs):
         super().__init__()
         self.tag = tag
+        self._link = tag
```

Comment:
Already set in widgetize (but my idea was the Changeable.__init__ will set this for every widget, I've written it alongside)